app/notifications/process_notifications.py

```diff
@@ -138,7 +138,7 @@ def send_notification_to_queue(notification, research_mode, queue=None):
     if notification.notification_type == LETTER_TYPE:
         if not queue:
             queue = QueueNames.CREATE_LETTERS_PDF
```

Comment:
Since we are changing the name of the task do we want to change the name of the queue?

Reply:
what would you change it to?